xcode/Subconscious/Shared/Models/Petname.swift

```diff
@@ -60,6 +60,12 @@ public struct Petname:
     public var verbatimMarkup: String {
         "@\(self.verbatim)"
     }
+    
+    public var parts: [Petname] {
```

Comment:
This is not feedback so much as a style question I want to raise to get your thoughts:

I tend to have a soft instinct to define dynamic properties when it "costs nothing". That is, when it does something extremely cheap, like looking up two properties, or string templating. If the operation is more complex, costly, or heap allocates, I often define a method. Also if the call is throwing or async, ofc.

I don't think the below should change. I'm mostly raising this to ask if we want to try to define a more consistent style guideline. Perhaps all zero-arg methods should be properties? Or maybe this is best left to the author to "feel what the vibe of this property is".

Reply:
I think I share your intuition in that I would _not_ expect a property to allocate memory every time I access it. 

I'm not sure if it's targeted enough as a heuristic but imo if accessing a given property it in a tight, critical loop is a footgun in hiding (expensive, memory alloc, side effects) then I would prefer a function?

I've updated `.parts` to a function in line that view.